### PR TITLE
[MIG] [11.0] web_sheet_full_width

### PR DESCRIPTION
**Pull request by Jerther**
Followup to https://github.com/OCA/web/pull/755

Had to make another pull request so commit history from 10.0 is preserved.

This one also includes changes requested by @Yajo .

### CODE REVIEW

File: web_sheet_full_width/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "version": "11.0.1.0.1",
+    "author": "Therp BV, Sudokeys, GRAP, Métal Sartigan, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "summary": "Use the whole available screen width when displaying sheets",
+    "category": "Tools",
+    "depends": [
+        'web',
+    ],
+    "data": [
+        "view/qweb.xml",
```
Comment (oihane) [inline web_sheet_full_width/__manifest__.py]: @Jerther I think you missed sth

**Comment by Jerther** [comment]
Now this is strange. If you look at the source branch:
https://github.com/metalsartigan/web/tree/Jerther-web_sheet_full_width
There is a new commit I just pushed, but it won't appear here... Any idea??

**Comment by Jerther** [comment]
Even weirder, the runbot caught the commit even if it's not in the PR
http://runbot.odoo-community.org/runbot/build/3307416
My other PR in `project` has been going well. I have no idea what's wrong with this branch. I could just delete it, start a fresh one, do it right and make another PR if that's not too much trouble for you guys.

**Comment by pedrobaeza** [comment]
I think it's something with GitHub. Sometimes it flickers...

**Review by yajo — APPROVED** [review]
Code OK

**Review by oihane — APPROVED** [review]
Code LGTM
Tested locally in ce and ee and it looks really good

**Comment by Jerther** [comment]
:tada:

**Pull request merged**